examples/firebase/index.js

```diff
@@ -0,0 +1,179 @@
+const functions = require("firebase-functions");
+const { PKPass } = require("passkit-generator");
+const admin = require("firebase-admin");
+var fs = require('file-system');
+var path = require('path');
+var axios = require('axios');
+var os = require('os');
+
+// Firebase init
+admin.initializeApp({
+    credential: admin.credential.cert(require("CERTIFICATE_PATH")),
+    storageBucket: "STORAGE_BUCKET_URL"
+});
+var storageRef = admin.storage().bucket()
+
+exports.pass = functions.https.onRequest((request, response) => {
+    PKPass.from({
+        // Get relevant pass model from model folder (see passkit-generator/examples/models/)
+        model: `./model/${request.body.passType}.pass`,
```

Comment:
Is model being read correctly on a cloud function?

Reply:
Yes that seems to work fine, essentially I have a folder which has everything in it, including a `model` folder containing some models I wrote based on the Apple Developer examples, each with the relevant name (e.g. boardingPass.pass, generic.pass etc.), and this entire folder gets deployed to Firebase. I think scriptpapi covers this in his video if I remember right? You might like have a look there if you'd like to see it in action.

Reply:
Same here, I'm wondering how it could read the files if Firebase cannot use FileSystem API 🤔 What is `./model` relative to? Scriptpapi is actually using model, but I'm not sure what am I losing. Maybe `fs` works actually? But I honestly don't see where it is going to take the files from through `fs`.

Reply:
Again I'm not sure, let me have another look at scriptpapi's video and try and figure out what's going on!

Reply:
As we were saying in another discussion above, maybe reading is fine but writing is illegal or something like this.

Anyway, maybe I'd not use the model as a folder in a Firebase cloud function, because it might mean a higher cold start time (see https://firebase.google.com/docs/functions/tips?hl=en#use_dependencies_wisely, even if it doesn't quote the File System).

I guess it is a tradeoff with the pricing. By looking [here](https://cloud.google.com/firestore/pricing?hl=en), for `eur3` (Europe region) - but maybe this is valid for all the regions - you have 50k free readings per day. This means that one might have a zip file and unzip it on runtime.

It depends on how many files you need to read, multiplied by the average number of passes you'll generate in a single day.

For the example purpose, using `fs` would be really enough, with a comment.

Or maybe two different examples should be created.